testsuite/Rakefile

```diff
@@ -90,6 +90,15 @@ namespace :utils do
     output_filepath = "#{args[:output_path]}/#{args[:client_name]}_#{File.basename(args[:template_path], '.*')}.feature"
     sh "sed 's/<client>/#{args[:client_name]}/g' #{args[:template_path]} > #{output_filepath}", verbose: false
     sh "sed -i 's/<type>/salt/g' #{output_filepath}", verbose: false
+    # When two minions with the same version but different architectures create a bootstrap repository, they will share the same directory
+    # To prevent one minion to delete the other minion bootstrap repository, disable the flush option for s390 minion.
+    if args[:flush]
+      # This will generate `minion_create_bootstrap_repository.feature` with the flush option active
```

Comment:
If we know that this only handled on `minion_create_bootstrap_repository` , I'd have another condition in that `if` statement, so we check if `output_filepath` is `minion_create_bootstrap_repository`, if so, we apply the `sed`, otherwise we don't need to apply this `sed` command for every single feature file running on that task.